SCT-Cpp.pro

```diff
@@ -1,10 +1,12 @@
 TEMPLATE = subdirs
 SUBDIRS  = \
 	SctCppTimer \
-	SctCppUnit
+	SctQtTimer \
+	SctUnitTimer
 
-SctCppTimer.file = SctCppTimer/SctCppTimer.pro
-SctCppUnit.file  = SctCppTimer/SctCppUnit.pro
+SctCppTimer.file  = SctCppTimer/SctCppTimer.pro
+SctQtTimer.file   = SctCppTimer/SctQtTimer.pro
```

Comment:
Builds, but gives us the known shadow and override warnings for the generated code.  
How about adding ``-Wshadow`` and ``-Wsuggest-override``  to the compiler flags?

Reply:
Hm... just found out, that those flags are already set. But why do i still get the warnings?

Reply:
Setting the compiler flags enables the warnings.